android/src/main/java/org/reactnative/camera/tasks/ResolveTakenPictureAsyncTask.java

```diff
@@ -76,7 +76,7 @@ protected WritableMap doInBackground(Void... voids) {
 
         // we need the stream only for photos from a device
         if (mBitmap == null) {
-            mBitmap = BitmapFactory.decodeByteArray(mImageData, 0, mImageData.length);
+            mBitmap = ImageCacheUtils.getResizedBitmap(mImageData, 400);
```

Comment:
Is this a height limit ? So we can't have pictures bigger than 400 ?

Reply:
should this be a prop?

Reply:
I think mImageData.length was correct and in that case it doesn't make sense this PR then.